### DISCUSSION TITLE
New UI of Paralink

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#175] [General]
Visit: https://paralink.darwinia.network/

Thanks for @nblogist and his team for supporting this. It will be great if the community can test and feedback.

There are known improvements/issues including more comprehensive wallets integration to be supported in the nexts, and please advice the chains and assets in Polkadot ecosystem you want to support.

**Comment by Divblaze** [comment]
Nice UI but not compatible with Mobile  wallet

**Reply by 2xJDubs** [reply]
Which mobile wallet are you trying.  Just tested on Metamask Mobile and it worked just fine.

**Reply by Divblaze** [reply]
Wallet connect feature not redirecting to wallet list

**Reply by 2xJDubs** [reply]
Odd, it worked for me.  What mobile wallet were you using?

**Reply by nblogist** [reply]
@2xJDubs do you have any feedback? I would appreciate you opening tickets on the repo [here](https://github.com/darwinia-network/paralink-ui)

**Reply by Divblaze** [reply]
I'll open a ticket with a screenshot of the issue am getting

**Comment by SasoLithops** [comment]
I see some up there already but for now I recommend adding ASTAR(ASTR, WETH, WBTC), MOONBEAM (GLMR), INTERLAY (iBTC)

**Reply by nblogist** [reply]
as in integrating more?
